app/views/roles/_past_role_holders.html.erb

```diff
@@ -0,0 +1,17 @@
+<% unless role.past_holders.empty? %>
+  <section id="past-role-holders" class="govuk-!-padding-bottom-9">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Past role holders:",
```

Comment:
```suggestion
      text: "Previous holders of this role",
```

This is what it says in Whitehall: https://www.gov.uk/government/ministers/parliamentary-under-secretary-of-state#previous-holders-of-this-role